updater/osv/osv.go

```diff
@@ -600,14 +619,26 @@ func (e *ecs) Insert(ctx context.Context, skipped *stats, name string, a *adviso
 				case `ECOSYSTEM`:
 					switch af.Package.Ecosystem {
 					case ecosystemMaven, ecosystemPyPI, ecosystemRubyGems:
+						vs.semverRange = nil
```

Comment:
should this be mirrored above to make `vs.ecosystemRanges = nil`?

Reply:
We can deduce that the `vs.ecosystemRanges` haven't been touched later by checking the `len()` we can't really do that with `vs.semverRange` so that's why its explicitly nil(l)ed